jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: classic-docs
+title: "API v2 Introduction"
+short-title: "Introduction"
+description: "introduction to the CircleCI API v2"
+categories: [getting-started]
+order: 1
+---
+
+The CircleCI API v2 is a significant update to the existing CircleCI API v1.1 that is currently being used by internal and external audiences.
+
+## Introduction to API v2
+
+CircleCI API v2 has significantly changed the current API v1.1 paradigm to incorporate some important changes and shifts in the way that you can and will use the API for your jobs. Although changes were made to the available endpoints (discussed later on this page), two of the most important changes are the introduction of "Pipelines" and "Project Slugs" as concepts in the API design and architecture, as well as a set of new endpoints that enable you to:
+
+- Retrieve a workflow by ID
+- Trigger Pipelines with parameters
+- Trigger particular workflows within Pipelines
+- Retrieve a Pipeline by ID
+- Retrieve recent Pipelines for an organization or for a project (ideally with branch filtering)
+
+**Note** In API v1.1, the term "build processing" was used to describe how projects were built and processed; however, with the API v2 release, "build processing" has been replaced by "pipelines." New projects will have "pipelines" set by default, whereas existing projects will not be changed.
+
+## Getting Started with the API v2
+
+The v2 API is very similar to the current v1.1 API with some notable exceptions.
+
+Here is a simple example using `curl` to trigger a pipeline with parameters 
+
+```
+curl -u ${CIRCLECI_TOKEN}: -X POST --header "Content-Type: application/json" -d '{
+  "parameters": {
+    "myparam": "./myspecialdir",
+    "myspecialversion": "4.8.2"
+  }
+}' https://circleci.com/api/v2/project/${project_slug}/pipeline
+```
+
+In the above example the `project_slug` would take the form :vcs/:org/:project. For example, the project slug `gh/CircleCI-Public/circleci-cli` tells `CircleCI` to use the project found in the GitHub organization CircleCI-Public in the repository named `circleci-cli`.
+
+### Authentication
+
+The CircleCI API v2 enables users to be authenticated in much the same way as they would be authenticated in the previous v1.1 API. A simple way to authenticate is to send your API token as the username of the HTTP request. For example, if you have set `CIRCLECI_TOKEN` in your shell's environment, then you could then use `curl` with that token like the example shown below:
+
+`curl -u ${CIRCLECI_TOKEN} https://circleci.com/api/v2/me`
+
+For more detailed information on how to authenticate with a token, please refer to the [Authentication](https://circleci.com/docs/api/#authentication) section in the main CirlceCI API documentation.
+
+### Pipelines
```

Comment:
We already have a page on Pipelines -- should we maybe link to that (and maybe update with some of the content here? Pipelines themselves seem like a general part of the system, not something that should be specially documented relative to the API.

Reply:
Removed all pipelines content and added link to Pipelines page